axelor-business-project/src/main/java/com/axelor/apps/businessproject/db/repo/ProjectTaskBusinessProjectRepository.java

```diff
@@ -18,10 +18,19 @@
  */
 package com.axelor.apps.businessproject.db.repo;
 
+import com.axelor.apps.businessproject.service.ProjectTaskProgressUpdateService;
 import com.axelor.apps.hr.db.repo.ProjectTaskHRRepository;
 import com.axelor.apps.project.db.ProjectTask;
+import com.google.inject.Inject;
 
 public class ProjectTaskBusinessProjectRepository extends ProjectTaskHRRepository {
+  ProjectTaskProgressUpdateService projectTaskProgressUpdateService;
+
+  @Inject
+  public ProjectTaskBusinessProjectRepository(
+      ProjectTaskProgressUpdateService projectTaskProgressUpdateService) {
+    this.projectTaskProgressUpdateService = projectTaskProgressUpdateService;
+  }
```

Comment:
In repository, we don't use constructor injection, we create a variable calling the service using Beans.get().